modules/bringauto_package/Package.go

```diff
@@ -113,13 +112,28 @@ func (packg *Package) CreatePackageName() string {
 	return strings.Join(packageName, "")
 }
 
+// Returns short package name without versions
+func (packg *Package) GetShortPackageName() string {
```

Comment:
Lib prefix is missing there. I would prefer to use GetShortPackageName in manner of a replacement.

For example 
for libgtest-dev_v1.12.1_x86-64-debian-11.zip the GetShortPackageName should return libgtest-dev.

Also GetShortPackageName should be used to construct full package name. (to avoid duplicities)

Reply:
Creating short package name moved to CreatePackageName method, which has new `short` parameter.

Reply:
Maybe do not use bool because the code is more readable with it.

Maybe use Enum instead of bool? Or split the function into two?
"Also GetShortPackageName should be used to construct full package name. (to avoid duplicities)" - maybe better phrasing would be "Should be used by GetPackageName function".

Reply:
Reworked as requested.